lib/jsdom/browser/parser/xml.js

```diff
@@ -51,7 +51,22 @@ function appendChild(parent, child) {
 }
 
 function createParser(rootNode, ownerDocument, saxesOptions) {
-  const parser = new SaxesParser(saxesOptions);
+  const parser = new SaxesParser({
+    ...saxesOptions,
+    //
+    // There are no situations under which it is legitimate to set the following
+    // parameters to any other value than the ones we set here.
```

Comment:
Could we be a bit clearer about what this means? Like what this option does, and why it should always be true.

Reply:
Ping @lddubeau for this comment in particular.

Reply:
Sorry for the delay. I'm spread thin these days. 

It is not just the one option that must always be true but all the following parameters that must always be set the way they are hard coded there. `xmlns: false` (which is the only other possible value for `xmlns`) would turn off namespace support and cannot be correct when parsing XML as a browser does because browsers don't parse XML without namespace support. The two other options must be set the way they are because browsers support only XML 1.0, not 1.1.

The comment is there to signify that it is not a mistake if these parameters are being hard-coded there because there's no legitimate scenario in jsdom under which they should be set differently.